src/util.js

```diff
@@ -188,7 +190,9 @@ const IMPORTANT_RE = /^([^:]+:.*?)( !important)?;$/;
 
 // Given a single style rule string like "a: b;", adds !important to generate
 // "a: b !important;".
+/* eslint-disable no-unused-vars */ // forgive existing code
 export const importantify = (string) =>
     string.replace(
         IMPORTANT_RE,
         (_, base, important) => base + " !important;");
+/* eslint-enable no-unused-vars */
```

Comment:
The unused variable is `important` from line 197:
`(_, base, important) => base + " !important;");`

I'll remove it instead of being too timid here. :)